### PR TITLE
remove broker support

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

File: sdk/identity/azure-identity/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@
 
 ### Breaking Changes
 
+> These changes do not impact the API of stable versions such as 1.14.0.
+> Only code written against a beta version such as 1.15.0b1 may be affected.
+- Windows Web Account Manager (WAM) Brokered Authentication is moved into another package.
```
Comment (pvaneck) [inline sdk/identity/azure-identity/CHANGELOG.md:13]: Should mention the name of the package
```suggestion
- Windows Web Account Manager (WAM) Brokered Authentication is moved into another package (`azure-identity-broker`)
```

**Review by pvaneck — COMMENTED** [review]
Don't forget to update browser credential removing parent_window_handle and msa_passthrough from `acquire_token_interactive`.

File: sdk/identity/azure-identity/CHANGELOG.md (hunk 1 above)
Comment (xiangyan99) [reply sdk/identity/azure-identity/CHANGELOG.md:13]: We have not finalized the package name yet. :)

Comment (pvaneck) [reply sdk/identity/azure-identity/CHANGELOG.md:13]: Ah good point

**Review by pvaneck — APPROVED** [review]

**Pull request merged**